roles/openshift_hosted_facts/tasks/main.yml

```diff
@@ -1,7 +1,22 @@
 ---
+- set_fact:
+    openshift_hosted_router_selector: "{{ openshift_hosted_infra_selector }}"
+  when: openshift_hosted_router_selector is not defined and openshift_hosted_infra_selector is defined
+- set_fact:
+    openshift_hosted_registry_selector: "{{ openshift_hosted_infra_selector }}"
+  when: openshift_hosted_registry_selector is not defined and openshift_hosted_infra_selector is defined
+- set_fact:
+    openshift_hosted_logging_selector: "{{ openshift_hosted_infra_selector }}"
+  when: openshift_hosted_logging_selector is not defined and openshift_hosted_infra_selector is defined
```

Comment:
openshift_hosted_{logging,metrics}_selector should probably be dropped from this PR.